senza/cli.py

```diff
@@ -1634,6 +1616,8 @@ def wait(stack_ref, region, deletion, timeout, interval):
             return
     raise click.Abort()
 
+cli.add_command(cmd_config)
```

Comment:
Yes, because [@command](https://github.com/zalando-stups/senza/pull/327/files#diff-a8d4b26c8d4f994b3beedd0f80a63485R10) only makes the function a command. `add_command` makes it a subcommand of another command.